File: Codigos Examen l prog/C++/Fibonacci.cpp
#include <iostream>

int main() {
    int n;
    std::cout << "Ingrese el número de términos (n): ";
    std::cin >> n;

    int a = 0;
    int b = 1;

    if (n >= 1) {
        std::cout << a << " ";
    }
    if (n >= 2) {
        std::cout << b << " ";
    }

    for (int i = 3; i <= n; ++i) {
        int temp = a + b;
        std::cout << temp << " ";
        a = b;
        b = temp;
    }
    std::cout << std::endl;

    return 0;
}

File: Codigos Examen l prog/Java/Fibonacci.java
import java.util.Scanner;

public class Fibonacci {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese el número de términos (n): ");
        int n = scanner.nextInt();

        int a = 0;
        int b = 1;

        if (n >= 1) {
            System.out.print(a + " ");
        }
        if (n >= 2) {
            System.out.print(b + " ");
        }

        for (int i = 3; i <= n; i++) {
            int temp = a + b;
            System.out.print(temp + " ");
            a = b;
            b = temp;
        }
        System.out.println();
        scanner.close();
    }
}